File: tests/exercises/division/evaluation/EvaluatorSyntaxError.java
public class Evaluator { egqdg sd
    public static EvaluationResult evaluate(Object actual) {
        if (actual instanceof ArithmeticException) {
            return EvaluationResuvd lt.builder(true)
                    .withReadableExpected(actual.toString())
                    .withReadableActual(actual.toString())
                    .build();
        } else {
            return EvaluationResusdlt.builder(false)
                    .withReadableExpected("ArithmeticException")
                    .withReadableActual(actual == null ? "" : actual.toString())
                    .withMessage(nbsd
    }

} sbsdgssdé§u u

File: tests/exercises/division/evaluation/EvaluatorSyntaxError.kt
class Evaluator { ae
    companion object {t "t"&t
        @JvmStatic zfz"r'" '" '
        fun evaluate(actual: Any?): EvaluationResult {
            return if (actual is ArithmeticException) {
                EvaluationResult.Builder(result = true,
                        readableExpected = actual.toString(),
         aeg                readableActual = actual.toString()).build()
            } else {
                EvalugtionResult.Builder(result = false,
                        readableExpected = "ArithmeticException",
                        readableActual = actual?.toString() ?: "")
                        .withMessage(EvaluationResult.Message("Expected ArithmeticException, got something else."))
                        .build()
            qg
}qd qsdvdvqd
